GitUI/CommandsDialogs/RevisionFileTreeControl.cs

```diff
@@ -257,7 +257,8 @@ public enum Command
             OpenAsTempFile = 3,
             OpenAsTempFileWith = 4,
             EditFile = 5,
-            FilterFileInGrid = 6
+            FilterFileInGrid = 6,
+            Find = 7,
```

Comment:
This Find is different from Find in RevDiff FileViewer

```suggestion
            FindFile = 7,
```

Reply:
Renamed in both RevisionDiffControl and RevisionFileTreeControl.

Other ShortcutKeys should be dealt with separately (if necessary).

Reply:
In RevDiff I expected the Find to search the file contents, the searchbox exists too.
The name should be changed to something similar as in FileTree (`Find in file tree`). Just `Find file`?
I also propose that Ctrl-F by default is bound to search.

But this is likely another PR.

Reply:
> In RevDiff I expected the Find to search the file contents, the searchbox exists too. The name should be changed to something similar as in FileTree (`Find in file tree`). Just `Find file`?

Yes, already done:
> >  Renamed in _both_ RevisionDiffControl and RevisionFileTreeControl.

>  I also propose that Ctrl-F by default is bound to search.
> But this is likely another PR.

No need for a PR, just unassign the `FindFile` Hotkey.
`Ctrl+F` should have activated the file search before (via shortcut in context menu) but the Hotkey took precedence.

Reply:
> No need for a PR, just unassign the FindFile Hotkey.

The change is to unassign by default...

Reply:
> The change is to unassign by default...

Yes.

@RussKie, I guess you did not miss `Ctrl+F` opening the file search in the `Diff` tab. So, I expect you will agree to not assign a default Hotkey for `RevisionDiffControl.Command.FindFile`, don't you?

Reply:
Yes, that's fine.